examples/java/src/main/java/org/apache/beam/examples/WordCount.java

```diff
@@ -166,7 +166,7 @@ public String apply(KV<String, Long> input) {
     /**
      * Returns "gs://${YOUR_TEMP_DIRECTORY}/counts.txt" as the default destination.
      */
-    public static class OutputFactory implements DefaultValueFactory<String> {
```

Comment:
Note for naive reviewers: "Classes inside of interfaces by definition are public and static, so the public and static modifiers on the inner classes are redundant. On the other hand, classes inside of interfaces can be abstract or non abstract. So, abstract modifier is allowed.: